DOCS-8935-remove-support-v4.3-cv (#1620)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/execution-engine.adoc

* If no custom threading settings are applied (either through `scheduler-pools.conf` or directly in the Mule app), then no action is required.
* If custom threading configurations are applied, test your Mule applications using the default configuration. +
Because of the UBER pool strategy and other performance updates implemented in Mule 4.3, you might not need custom threading configurations.

> * If tests confirm that a custom configuration is still necessary, it might indicate that there is a problem with resource management in the application, one of the application dependencies, or the Mule instance.

Reviewer: ```suggestion
* If tests confirm that a custom configuration is still necessary, a problem might exist with resource management in the application, one of the application dependencies, or the Mule instance.
```

Reviewer: Or "necessary, check for problems with..."
